scipy/spatial/transform/_rigid_transformation.pyx

```diff
@@ -846,6 +846,20 @@ cdef class RigidTransformation:
         This implements the exponential map that converts 6-dimensional real
         vectors to SE(3).
 
+        An exponential coordinate vector consists of 6 elements - 3 for rotation (known
+        as a rotation vector) and 3 for translation. The exponential mapping can be
+        expressed as matrix exponential ``M = exp(m)``. Where ``M`` is a 4x4 matrix
```

Comment:
I would try to use symbols that are used in the literature more often. For transformation matrices we agreed on $t$ for the documentation I guess, but I would suggest to use $\boldsymbol{\tau}$ for exponential coordinates.

In the literature I found

* $\boldsymbol{\tau}$, e.g., http://www.iri.upc.edu/files/scidoc/2089-A-micro-Lie-theory-for-state-estimation-in-robotics.pdf (although they use a different order of the orientation and translation in the vector!)
* $\boldsymbol{S} \theta$, e.g., https://link.springer.com/book/10.1007/978-3-319-54413-7
* $\mathcal{S} \theta$, e.g., https://hades.mech.northwestern.edu/index.php/Modern_Robotics
* $\boldsymbol{\delta}, e.g., https://ethaneade.com/lie.pdf

But maybe you know other conventions?

Reply:
My notation is sketchy, I admit. 

There are two approaches: use simple text notation to try and fit "formulas" into the main section or use proper TeX, but put it into Notes section.

I came to understanding that the first approach is preferable, if you can get away with it. Sort of keep things simple and readable in any environment, TeX only makes good sense rendered in html.

Here I feel like it is very borderline. For me it is useful to know how technically this exponentiation done, for others it may be an unnecessary detail. 

If we want to keep everything in a simplified text format, how can I improve the notation? I wanted to use very basic one-letter symbols.

Reply:
~~I would argue that M should be renamed to t, since that is what we use in the rest of the documentation for transformation matrices.~~ Ok, just noticed that that's not true. I think t is for objects of the RigidTransformaation class.

I've seen T or H for transformation matrices in the literature, so maybe T is the best option here, since tau would also be the Greek t. Then we use tau for the 4x4 log matrix? Or is tau for the vector and [tau] for the matrix an option?

Reply:
I've used `T` for the transform matrix and `tau` for the matrix formed from se3 vector.